### PR TITLE
CVE-2010-3118 and CVE-2016-5153

### PR DESCRIPTION
**Pull request by ahartnett**

### CODE REVIEW

**Comment by ahartnett** [comment]
CVE-2010-3118 and CVE-2016-5153

File: cves/CVE-2016-5153.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 CVE: CVE-2016-5153
-CWE: 
+CWE: Catagory 19 Data Handling 
 announced: 2016-09-11 06:59:08.943000000 -04:00
-description_instructions: |
+description_instructions:
```
Comment (BenWoodworth) [inline cves/CVE-2016-5153.yml]: The pipe character tells YAML the text is multi-line, and starts on the next line.

File: cves/CVE-2010-3118.yml [hunk 2]
```diff
@@ -15,7 +15,9 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: The implementation of the "omnibox" feature on Chrome did not anticipate
```
Comment (nns3455) [inline cves/CVE-2010-3118.yml]: Use a pipe | for this, like how it's done with description_instructions at line 5

File: cves/CVE-2016-5153.yml [hunk 3]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: Blink a tool used in chrome to support web animations improperly 
+relies on the use on list iteration. Websites that use this tool open themselves
+up to denial of service (use-after-destruction) style attacks or possibly have 
+unspecified other impact to the website.
```
Comment (BenWoodworth) [inline cves/CVE-2016-5153.yml]: Change to this format so the YAML parses correctly:
```
description: |
  Blink a tool [...]
  relies on the [...]
  [...]
```

File: cves/CVE-2016-5153.yml [hunk 4]
```diff
@@ -75,20 +78,21 @@ discovered:
...
   question: |
     What subsystems was the mistake in?
 
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer: 
-  name: 
+  answer: The path that this bug was founs was in the path /cc/animations so it
+  is assumed that it is in the animation subsystem
```
Comment (BenWoodworth) [inline cves/CVE-2016-5153.yml]:
```
answer: |
  The path [...]
  [...]
```

File: cves/CVE-2016-5153.yml [hunk 5]
```diff
@@ -143,8 +150,9 @@ lessons:
     applies: 
     note: 
   distrust_input:
-    applies: 
-    note: 
+    applies: true
+    note: by relying on a linked list to fuction an attacker could use this web animations
+    service to inject malcious code into a victims computer
```
Comment (BenWoodworth) [inline cves/CVE-2016-5153.yml]:
```
note: |
  by relying [...]
  [...]
```

File: cves/CVE-2016-5153.yml [hunk 6]
```diff
@@ -175,4 +183,5 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: In my opinion this is a coding mistake, it is bad practice to rely on
+  linked list in order to function properly.
```
Comment (BenWoodworth) [inline cves/CVE-2016-5153.yml]:
```
answer: |
  In my opinion [...]
  [...]
```

File: cves/CVE-2010-3118.yml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 CVE: CVE-2010-3118
-CWE: 
+CWE: Catagory 200 Information Leak / Disclosure
```
Comment (andymeneely) [inline cves/CVE-2010-3118.yml]: Just the number is fine. I would also say that there's probably a more specific CWE that applies here. Look at the CWE ParentOf relationships on the 200 entry and you'll probably find a more appropriate one.

File: cves/CVE-2016-5153.yml [hunk 8]
```diff
@@ -1,8 +1,8 @@
 ---
 CVE: CVE-2016-5153
-CWE: 
+CWE: Catagory 19 Data Handling
```
Comment (nns3455) [inline cves/CVE-2016-5153.yml]: Only include the number:
`CWE: 19`

File: cves/CVE-2010-3118.yml (hunk 2 above)
Comment (andymeneely) [inline cves/CVE-2010-3118.yml]: Interesting! Sounds like a requirements issue, actually. They didn't consider that flow to begin with. I could also use more details on what steps a user would take to copy their password into the ominbox, and in what situations they might do that.

File: cves/CVE-2010-3118.yml (hunk 7 above)
Comment (nns3455) [inline cves/CVE-2010-3118.yml]: Only include the number:
`CWE: 200`

**Pull request closed without merging**